packages/website/src/content/docs/about.mdx

```diff
@@ -7,7 +7,7 @@ sidebar:
   order: 0
 ---
 
-Spotlight started out as a Hackweek project at Sentry:
+Spotlight started out as a [Hackweek project](https://syntax.fm/show/666/hackweek-projects-realtime-markdown-editor-and-a-hardware-recording-button) at Sentry:
```

Comment:
Find it odd that links to a project that's unrelated to spotlight though

Reply:
Fair - I think this is the only recent public facing record we have for what Hackweek is. 

They should interview David about Spotlight 😏